src/Compilers/CSharp/Test/Semantic/Semantics/SimpleProgramsTests.cs

```diff
@@ -1099,7 +1306,6 @@ void local2()
 ";
 
             var comp = CreateCompilation(new[] { text1, text2, text3 }, options: TestOptions.DebugExe, parseOptions: DefaultParseOptions);
-            comp.VerifyDiagnostics();
```

Comment:
Would it make sense to also test a compilation made from `text1 + text2 + text3`?

Reply:
> Would it make sense to also test a compilation made from text1 + text2 + text3?

I do not think so

---
In reply to: [397467331](https://github.com/dotnet/roslyn/pull/42648#discussion_r397467331) [](ancestors = 397467331)